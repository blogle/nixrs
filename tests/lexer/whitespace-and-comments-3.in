

	
 # comments and
 /* whitespace
 don't /* matter */



