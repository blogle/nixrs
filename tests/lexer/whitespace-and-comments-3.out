[4:2-4:16] (Comment) "# comments and"
[5:2-6:20] (Comment) "/* whitespace\n don\'t /* matter */"
